workflows/sarek.nf

```diff
@@ -58,6 +60,27 @@ if (params.wes) {
     if (params.intervals && !params.intervals.endsWith("bed") && !params.intervals.endsWith("interval_list")) exit 1, "Interval file must end with .bed or .interval_list"
 }
 
+if(params.tools && params.tools.contains('ascat')){
+    if(!params.ascat_alleles){
+        log.error "No allele files were provided for running ASCAT. Please provide a zip folder with allele files."
+        exit 1
+    }
+    if(!params.ascat_loci){
+        log.error "No loci files were provided for running ASCAT. Please provide a zip folder with loci files."
+        exit 1
+    }
+    if(!params.ascat_loci_gc && !params.ascat_loci_rt){
+        log.warn("No LogRCorrection performed in ASCAT.")
+    }
+    if(params.wes){
+        log.warn("Reference files not suited for running ASCAT on WES data. It's recommended to use the reference files provided here: https://github.com/Wedge-lab/battenberg#required-reference-files")
```

Comment:
This is printed everytime wes is selected
```suggestion
        log.warn("Default reference files not suited for running ASCAT on WES data. It's recommended to use the reference files provided here: https://github.com/Wedge-lab/battenberg#required-reference-files")
```

Reply:
Yes, this should actually be printed if `params.wes` is true until we have prepared/uploaded specific reference files for `wes`.